src/main/java/RacingCarApplication.java

```diff
@@ -12,22 +13,24 @@ public static void main(String[] args) {
         InputDto input = inputView.getInput();
         ResultView resultView = new ResultView();
 
-        ArrayList<RacingCar> carList = new ArrayList<>();
+        ArrayList<String> nameList = input.getNameList();
+        ArrayList<RacingCar> carList = nameList
```

Comment:
타입을 유연하게 가져가는건 어떨까요? 🙂 List의 구현체를 이후에 LinkedList로 바뀌더라도 타입은 같으니 타입의 수정없이 변경할 수 있습니다!
```suggestion
        List<String> nameList = input.getNameList();
        List<RacingCar> carList = nameList
```